man/nvim.1

```diff
@@ -0,0 +1,411 @@
+.It Fl H
+Hebrew mode.
+Sets the options 'hkmap' and 'rightleft'.
+.It Fl V Ns Oo Ar N Oc Ns Op Ar file
+Verbose.
+Print messages about which files are being sourced and for reading and
+writing an nviminfo file.
+.Ar N
+is the value for the 'verbose' option; defaults to
+.Cm 10
+if omitted.
+If
+.Ar file
+is specified, append messages to
+.Ar file
+instead of printing them.
+.It Fl D
+Debugging mode.
+Started when executing the first command from a script.
+.It Fl n
+Disable the use of swap files.
+Sets the option 'updatecount' to
+.Cm 0 .
+Can be useful for editing file(s) on a slow medium.
+.It Fl r , L
+List swap files with information about recovering them.
+.It Fl r Ar file
+Recovery mode.
+The swap file
+.Ar file
+is used to recover a crashed session.
+The swap file has the same filename as the text file, but with ".swp" appended.
+See
+.Ic :help recovery .
+.It Fl u Ar nvimrc
+Use
+.Ar nvimrc
+instead of the default of
+.Pa ~/.nvimrc .
+If
+.Ar nvimrc
+is
+.Cm NORC ,
+do not load any initialization files (excluding plugins),
+and do not attempt to parse environment variables.
+If
+.Ar nvimrc
+is
+.Cm NONE ,
+loading plugins is also skipped.
+See
+.Ic :help initialization .
+.It Fl i Ar nviminfo
+Use
+.Ar nviminfo
+instead of the default of
+.Pa ~/.nviminfo .
+If
+.Ar nviminfo
+is
+.Cm NONE ,
+do not read or write an nviminfo file.
+See
+.Ic :help viminfo .
+.It Fl -noplugin
+Skip loading plugins.
+Implied by
+.Cm -u NONE .
+.It Fl o Ns Op Ar N
+Open
+.Ar N
+windows stacked horizontally.
+If
+.Ar N
+is omitted, open one window for each file.
+If
+.Ar N
+is less than the number of file arguments, allocate windows for the first
+.Ar N
+files and hide the rest.
+.It Fl O Ns Op Ar N
+Open
+.Ar N
+windows stacked vertically.
+If
+.Ar N
+is omitted, open one window for each file.
+If
+.Ar N
+is less than the number of file arguments, allocate windows for the first
+.Ar N
+files and hide the rest.
+.It Fl p Ns Op Ar N
+Open
+.Ar N
+tab pages.
+If
+.Ar N
+is omitted, open one tab page for each file.
+If
+.Ar N
+is less than the number of file arguments, allocate tab pages for the first
+.Ar N
+files and hide the rest.
+.It Cm + Ns Op Ar linenum
+For the first file, position the cursor on line
+.Ar linenum .
+If
+.Ar linenum
+is omitted, position the cursor on the last line of the file.
+Note that
+.Cm +5
+and
+.Cm -c 5
+on the command-line are equivalent to
+.Ic :5
+inside
+.Nm .
+.It Cm +/ Ns Op Ar pattern
+For the first file, position the cursor on the first occurrence of
+.Ar pattern .
+If
+.Ar pattern
+is omitted, the most recently used search pattern is used (if there is one).
+Note that
+.Cm /foo
+and
+.Cm -c /foo
+on the command-line are equivalent to
+.Ic /foo
+and
+.Ic :/foo
+inside
+.Nm .
+See
+.Ic :help search-pattern .
+.It Fl -cmd Ar command
+Execute
+.Ar command
+before processing any nvimrc.
+Up to 10 instances of these can be used, independently from instances of
+.Fl c .
+.It Fl c Ar command
+Execute
```

Comment:
It should be explained that `+command` works the same way as `-c command`, so maybe add `+command` in the left column. From the lines above it might seem that the '+' only works with linenumbers and search patterns